<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/5.png</normaloff>images/5.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>360</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_add">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/001.GIF</normaloff>../LSOonlienmusicplayer/images/001.GIF</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_play">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/620.bmp</normaloff>../LSOonlienmusicplayer/images/620.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/622.bmp</normaloff>../LSOonlienmusicplayer/images/622.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_stop">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/624.bmp</normaloff>../LSOonlienmusicplayer/images/624.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pre">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/mute.bmp</normaloff>../LSOonlienmusicplayer/images/mute.bmp</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_next">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/volumn.bmp</normaloff>../LSOonlienmusicplayer/images/volumn.bmp</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="horizontalSlider_Palygrogress">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_St">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color :rgb(255,255,255);
backfround-color:rgba(255,255,255,50);</string>
       </property>
       <property name="text">
        <string>动手输入</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../LSOonlienmusicplayer/images/620.bmp</normaloff>../LSOonlienmusicplayer/images/620.bmp</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="text">
        <string>成都-赵雷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_searchSongs">
       <property name="styleSheet">
        <string notr="true">color :rgb(255,255,255);
backfround-color:rgba(255,255,255,50);</string>
       </property>
       <property name="text">
        <string>search...</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../LSOonlienmusicplayer/images/001.GIF</normaloff>../LSOonlienmusicplayer/images/001.GIF</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_yes">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>41</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../LSOonlienmusicplayer/images/volumn.bmp</normaloff>../LSOonlienmusicplayer/images/volumn.bmp</iconset>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>441</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>在线音乐列表</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_singlist">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>142</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_About">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="image.qrc">
        <normaloff>:/new/prefix1/images/5.png</normaloff>:/new/prefix1/images/5.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_Close">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../LSOonlienmusicplayer/images/5.png</normaloff>../LSOonlienmusicplayer/images/5.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_yes_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_playerTime">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_Volumn">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>horizontalSlider_Palygrogress</zorder>
  <zorder>groupBox</zorder>
  <zorder>pushButton_yes</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>layoutWidget_3</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
